docs/server-workflow-search.md

```diff
@@ -109,184 +110,63 @@ There are some pre-allowlisted search attributes that are handy for testing:
-- Status is a mapping of int to state:
-  - 0 = unknown
-  - 1 = running
-  - 2 = completed
-  - 3 = failed
-  - 4 = canceled
-  - 5 = terminated
-  - 6 = continuedasnew
-  - 7 = timedout
-- StartTime, CloseTime and ExecutionTime are stored as INT, but support queries using both EpochTime in nanoseconds, and string in RFC3339 format (ex. "2006-01-02T15:04:05+07:00")
-- CloseTime, HistoryLength are only present in closed Workflow
-- ExecutionTime is for Retry/Cron user to query a Workflow that will run in the future
-
-To list only open Workflows, add `CloseTime = missing` to the end of the query.
-
-If you use retry or the cron feature to query Workflows that will start execution in a certain time range, you can add predicates on ExecutionTime. For example: `ExecutionTime > 2019-01-01T10:00:00-07:00`. Note that if predicates on ExecutionTime are included, only cron or a Workflow that needs to retry will be returned.
-
-### General Notes About Queries
-
-- Pagesize default is 1000, and cannot be larger than 10k
-- Range query on Temporal timestamp (StartTime, CloseTime, ExecutionTime) cannot be larger than 9223372036854775807 (maxInt64 - 1001)
-- Query by time range will have 1ms resolution
-- Query column names are case sensitive
-- ListWorkflow may take longer when retrieving a large number of Workflows (10M+)
-- To retrieve a large number of Workflows without caring about order, use the ScanWorkflow API
-- To efficiently count the number of Workflows, use the CountWorkflow API
-
-## Tools Support
-
-### CLI
-
-Support for search attributes is available as of version 0.6.0 of the Temporal server. You can also use the CLI from the latest [CLI Docker image](https://hub.docker.com/r/temporalio/tctl) (supported on 0.6.4 or later).
-
-#### Start Workflow with Search Attributes
+4. From the Docker output log, make sure Elasticsearch and Temporal started correctly.
+If you encounter an "insufficient disk space error", try running the following command:
 
 ```bash
-tctl --ns samples-namespace workflow start --tq helloWorldGroup --wt main.Workflow --et 60 --dt 10 -i '"vancexu"' -search_attr_key 'CustomIntField | CustomKeywordField | CustomStringField |  CustomBoolField | CustomDatetimeField' -search_attr_value '5 | keyword1 | vancexu test | true | 2019-06-07T16:16:36-08:00'
+docker system prune -a --volumes
 ```
 
-#### Search Workflows with List API
+5. Register a local Namespace and start using it.
 
-```bash
-tctl --ns samples-namespace wf list -q '(CustomKeywordField = "keyword1" and CustomIntField >= 5) or CustomKeywordField = "keyword2"' -psa
 ```
-
-```bash
-tctl --ns samples-namespace wf list -q 'CustomKeywordField in ("keyword2", "keyword1") and CustomIntField >= 5 and CloseTime between "2018-06-07T16:16:36-08:00" and "2019-06-07T16:46:34-08:00" order by CustomDatetimeField desc' -psa
+tctl --ns samples-namespace n re
```

Comment:
Wow is the command really "n re"?

Reply:
these are aliases. lets use full names in the docs:
```suggestion
tctl --namespace samples-namespace namespace register
```

Reply:
But actually why? Let's use `default` namespace and remove this all together.